openeogeotrellis/deploy/batch_job.py

```diff
@@ -356,11 +356,10 @@ def run_job(
                 ml_model_metadata = result.get_model_metadata(str(output_file))
                 logger.info("Extracted ml model metadata from %s" % output_file)
             for name, asset in the_assets_metadata.items():
-                # TODO: test in separate branch
-                # if not asset.get("href").lower().startswith("s3:/"):
-                #     # fusemount could have some delay to make files accessible, so poll a bit:
-                #     asset_path = get_abs_path_of_asset(asset["href"], job_dir)
-                #     wait_till_path_available(asset_path)
+                if not asset.get("href").lower().startswith("s3:/"):
```

Comment:
`asset.get("href")` could return None which will fail hard, so I would harden that with:

```suggestion
                if not asset.get("href", "").lower().startswith("s3:/"):
```

Reply:
and instead of skipping this arbitrary when `startswith("s3:/")`,
isn't it more robust to only do it the "href" looks like a local path: e.g. it starts with a slash  or a `file:/` protocol.